integration/wc-integration/src/main/java/se/inera/intyg/rehabstod/integration/wc/service/WcRestIntegrationServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.rehabstod.integration.wc.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.client.RestTemplate;
+import se.inera.intyg.rehabstod.integration.wc.service.dto.UnansweredCommunicationRequest;
+import se.inera.intyg.rehabstod.integration.wc.service.dto.UnansweredCommunicationResponse;
+
+import java.util.List;
+
+public class WcRestIntegrationServiceImpl implements WcRestIntegrationService {
+
+    @Value("${wc.service.logicalAddress}")
+    private String wcUrl;
+
+    @Value("${wc.getadditions.max.age.days:90}")
+    private int maxDaysOfUnansweredCommunication;
+
+    private final RestTemplate restTemplate;
+
+    public WcRestIntegrationServiceImpl(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    @Override
+    public UnansweredCommunicationResponse getUnansweredCommunicationForPatients(List<String> patientIds) {
+        final var url = wcUrl + "/internalapi/unanswered-communications";
+
+        final var request = new UnansweredCommunicationRequest();
+        request.setPatientIds(patientIds);
+        request.setMaxDaysOfUnansweredCommunication(maxDaysOfUnansweredCommunication);
+
+        return restTemplate.postForObject(url, request, UnansweredCommunicationResponse.class);
```

Comment:
Consider catching any errors and return an UnansweredCommunicationResponse with unansweredCommunicationError = true.